packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton.tsx

```diff
@@ -31,15 +34,43 @@ export const RainbowKitCustomConnectButton = () => {
 
               if (chain.unsupported || chain.id !== configuredNetwork.id) {
                 return (
-                  <>
-                    <span className="text-xs" style={{ color: networkColor }}>
-                      {configuredNetwork.name}
-                    </span>
-                    <button className="btn btn-sm btn-error ml-2" onClick={openChainModal} type="button">
+                  <div className="dropdown dropdown-end">
+                    <button tabIndex={0} className="btn btn-error btn-sm dropdown-toggle">
                       <span>Wrong network</span>
                       <ChevronDownIcon className="h-6 w-4 ml-2 sm:ml-0" />
                     </button>
-                  </>
+                    <ul tabIndex={0} className="dropdown-content menu p-2 shadow bg-base-100 rounded-box w-56">
+                      <li>
+                        <a
```

Comment:
We should use `button` here instead of `a` because it changes the URL to `/#` due to href every time users click it

Reply:
Thx for the review! 🙌 I've just pushed a commit with those tweaks:
- Changed both `a` for `button`
- Removed comments and `className="text-red-500"` from the childs, leaving it in parent element

Reply:
I should have taken a look on the code and see there was already a `button` for this purpose instead of an `a`, sorry 😅
Fixed now!